pkg/accounting/pnl/avg_cost.go

```diff
@@ -89,9 +86,9 @@ func (c *AverageCostCalculator) Calculate(symbol string, trades []types.Trade, c
 
 	return &AverageCostPnlReport{
 		Symbol:       symbol,
-		StartTime:    c.StartTime,
 		CurrentPrice: currentPrice,
 		NumTrades:    len(trades),
+		StartTime:    trades[0].Time,
```

Comment:
Need to check the length of `trades`?
If its length is zero, how about just early returning from this function?

Reply:
good point, let me fix it.